src/preamble.js

```diff
@@ -978,12 +978,33 @@ function createWasm(env) {
     return value;
   };
 #endif
+var proxyHandler = {
+  'get': function(obj, prop) {
+    if (prop in obj) {
+      return obj[prop]; // already present
+    }
+
+    // Check if the minification mapping exists first, if not just
+    // use the original name.
+    var funcName = prop;
+    if (Module["mapping"] && Module["mapping"][prop]) {
```

Comment:
btw, it would be better to not put this on `Module`. It can be just `var envMapping` at the global scope. That would allow better minification.

(Unless this does need to be exported to the outside for some reason?)